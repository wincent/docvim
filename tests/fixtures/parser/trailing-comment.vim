let g:counter=1 " Set counter to initial value.
