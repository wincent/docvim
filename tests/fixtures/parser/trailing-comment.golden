Unit [ LetStatement { letLexpr = "g:counter" , letValue = "1" } ]
